### ISSUE TITLE
Distinguish between task and workflow executions in the UI

### ISSUE THREAD

**Issue by katrogan** [flyteorg/flyte#3435]
From https://github.com/flyteorg/flyte/issues/3048

**Comment by kumare3** [comment]
@katrogan can we show in the Ui the user error vs system error bot that is set in the back

**Comment by katrogan** [comment]
hey @kumare3 what do you mean by "bot that is set in the back"?

**Comment by kumare3** [comment]
My bad - what I meant is that we have these fields already available in the database right?

**Comment by katrogan** [comment]
Yes we do! We have the reference launch plan or task that was used to launch the workflow
